tests/primer/primer_tool.py

```diff
@@ -112,11 +130,79 @@ def _handle_run_command(self) -> None:
         ) as f:
             json.dump(packages, f)
 
+    def _handle_compare_command(self) -> None:
+        with open(self.config.base_file, encoding="utf-8") as f:
+            main_dict: PackageMessages = json.load(f)
+        with open(self.config.new_file, encoding="utf-8") as f:
+            new_dict: PackageMessages = json.load(f)
+
+        final_main_dict: PackageMessages = {}
+        for package, messages in main_dict.items():
+            final_main_dict[package] = []
+            for message in messages:
+                try:
+                    new_dict[package].remove(message)
+                except ValueError:
+                    final_main_dict[package].append(message)
+
+        self._create_comment(final_main_dict, new_dict)
+
+    def _create_comment(
+        self, all_missing_messages: PackageMessages, all_new_messages: PackageMessages
+    ) -> None:
+        comment = ""
+        for package, missing_messages in all_missing_messages.items():
+            new_messages = all_new_messages[package]
+            package_data = self.packages[package]
+
+            if not missing_messages and not new_messages:
+                continue
+
+            comment += f"\n\n**Effect on [{package}]({self.packages[package].url}):**\n"
+
+            if missing_messages:
+                comment += "The following messages are no longer emitted:\n"
+            count = 1
+            for message in missing_messages:
+                comment += (
+                    f"{count}) {message['symbol']} on line {message['line']} here:\n"
+                )
+                filepath = str(message["path"]).replace(
+                    str(package_data.clone_directory), ""
+                )
+                comment += f"{package_data.url}/blob/{package_data.branch}/{filepath}#L{message['line']}\n"
```

Comment:
```suggestion
                filepath = str(message["path"]).replace(
                    str(package_data.clone_directory), ""
                )
                comment += (
                    f"{count}) [{message['symbol']} on line {message['line']}]("
                     f"{package_data.url}/blob/{package_data.branch}/{filepath}#L{message['line']})"
                )
```
Not sure if I did that right but the Intent is to have 

1. [locally-disabled on line 91](https://github.com/PyCQA/astroid/blob/main//astroid/__init__.py#L91)

It seems there's a problem with some links that lack the L before the line

Reply:
I fixed the `L`. I would keep the two lines for now as (at some point) we want to change the second line to an actual code snippet instead of just the link.

Reply:
Code snippet sure would be nice, but can't we do that by accessing the local version of the code we just linted ?

Reply:
Something like this -5 / +5 lines:
```python
# It's impossible to import from astroid.nodes with a wildcard, because
# there is a cyclic import that prevent creating an __all__ in astroid/nodes
# and we need astroid/scoped_nodes and astroid/node_classes to work. So
# importing with a wildcard would clash with astroid/nodes/scoped_nodes
# and astroid/nodes/node_classes.
from astroid.nodes import (  # pylint: disable=redefined-builtin (Ellipsis)
    CONST_CLS,
    AnnAssign,
    Arguments,
    Assert,
    Assign,
```

Reply:
We don't load the packages when making the comment. As I said in OP: we might be able to add the code snippets in the artefacts or download them in the comment workflow. But let's do that in a follow up.

Reply:
Shouldn't we do the comment at the same time we lint then ? I agree it's mergeable as is, but the decision to separate the check/message is taken now and would have to be refactored later.

Reply:
No we can't do that at the same time. See the comment at the top of this workflow file.

We need a different workflow trigger for the comment PR as it needs more permissions than you get when using `pull_request` as trigger. This is also why commenting and priming is separated on `mypy` and `pydocstringformatter`.

Reply:
All right, thanks for explaining :)

Reply:
I don't think it's worth adding a code snippet; linking to the github view better anyway as it's interactive. (Next best thing to actually loading the project in an IDE).